Wikipedia/Code/WMFRandomSectionController.h

```diff
@@ -5,6 +5,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+static NSString* const WMFRandomSectionIdentifier = @"WMFRandomSectionIdentifier";
```

Comment:
typically public constants are `extern` so as to encapsulate their raw value (and remove temptation to use it directly).